packages/db/src/schema/user.ts

```diff
@@ -10,6 +10,12 @@ export const user = mysqlTable("user", {
     sensorId: varchar("sensor_id", { length: 30 }).notNull(),
 });
 
+export const token = mysqlTable("token", {
```

Comment:
Verstehe ich das hier richtig, dass du für die Tokens eine neue DB-Tabelle anlegen willst? Dann müsstest du das ja auch noch in Planetscale umsetzen